src/site/stages/build/tome-sync-page-builder.js

```diff
@@ -0,0 +1,159 @@
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * This assumes the tome-sync output is sibling to the vets-website
+ * directory.
+ */
+const contentDir = path.join(
+  __dirname,
+  '../../../../../cms-tome-sync-export/content/',
+);
+
+/**
+ * When reading through entity properties, ignore these.
+ */
+const blackList = new Set([
+  'type',
+  'revision_uid',
+  'revision_user',
+  'user_id',
+  'items',
+  'owner_id',
+  'parent',
+  'role_id',
+  'roles',
+  'uid',
+  'vid',
+  'access_scheme',
+  'bundle',
+  // Temporarily ignore the following properties because they were
+  // causing circular dependencies. Once we get reader functions based
+  // on individual node / entity types, we can remove these from here.
+  // See the jsdoc on getEntityProperties for more information.
+  'field_facility_location',
+  'field_regional_health_service',
+  'field_region_page',
+  'field_office',
+]);
+
+/**
+ * Use to consistently reference to an entity.
+ *
+ * @param {String} type - The type of entity; corresponds to the file
+ *                        name.
+ * @param {String} uui  - The uuid of the entity; corresponds to the
+ *                        file name.
+ */
+const id = (type, uuid) => `${type}.${uuid}`;
+
+/**
+ * Take the entityName, return the contents of the file.
+ *
+ * Later, we can keep a counter for how many times we open a
+ * particular file to see if we can gain anything from caching the
+ * contents.
+ * @param {String} entityType - The type of entity; corresponds to the
+ *                              name of the file.
+ * @param {String} uuid - The UUID of the entity; corresponds to the
+ *                        name of the file.
+ */
+const fetchEntity = (type, uuid) =>
+  JSON.parse(
+    fs
+      .readFileSync(path.join(contentDir, `${type}.${uuid}.json`))
+      .toString('utf8'),
+  );
+
+/**
+ * Takes the type of entity and entity itself, returns an array of
+ * properties that we want to keep for this entity.
+ *
+ * @param {String} entityType - The type of entity; corresponds to the
+ *                              name of the file.
+ * @param {Object} entity - The contents of the entity itself before
+ *                          reference expansion.
+ *
+ * @return {Array<String>} - A list of properties of the entity to
+ *                           keep.
+ */
+const getEntityProperties = (entityType, entity) =>
+  // TODO: Change this to check the entity type and subtype and return
+  // a customized list.
+  Object.keys(entity);
```

Comment:
@bkjohnson I went ahead and pulled this out into its own function now to make it easier to add to later.